tests/generate-board-outline-gerber.test.ts

```diff
@@ -6,7 +6,7 @@ import {
 } from "src/stringify-gerber"
 import { maybeOutputGerber } from "tests/fixtures/maybe-output-gerber"
 import gerberToSvg from "gerber-to-svg"
-
+import { toMatchGerberSnapshot } from "./fixtures/preload"
```

Comment:
you don't need to import this, it should be registered on the global expect object